<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eisagogi</class>
 <widget class="QDialog" name="Eisagogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>337</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ΕΙΣΑΓΩΓΗ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background:#e3d7f0;
}
.QPushButton{
	border-radius: 8px;
	background:#557698;
	color:white;
}

.QPushButton:hover{
	background:#5696b6;
}

#onoma_l{
	font-size:12px;
	color:#193c69;
}

#mm_l{
	font-size:12px;
	color:#193c69;
}

#poso_l{
	font-size:12px;
	color:#193c69;
}


#arithomos_l{
	font-size:12px;
	color:#193c69;
}

#exp_l{
	font-size:12px;
	color:#193c69;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="new_med">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Νέο *</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="katahorisi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Καταχώριση</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLineEdit" name="onoma">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Όνομα</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="katigoria">
       <property name="placeholderText">
        <string>Κατηγορία</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
      <widget class="QLabel" name="arithmos_label">
       <property name="text">
        <string>Αριθμός</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" alignment="Qt::AlignTop">
      <widget class="QSpinBox" name="arithmos"/>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_3"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="barcode">
       <property name="placeholderText">
        <string>Barcode</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ημ. Λήξης</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QDateEdit" name="expday">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2020</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="mm">
       <property name="placeholderText">
        <string>ΜΜ</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" alignment="Qt::AlignTop">
      <widget class="QSpinBox" name="posotita">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
      <widget class="QLabel" name="poso_label">
       <property name="text">
        <string>Ποσότητα</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="category_cb"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="message">
     <property name="text">
      <string>Συμπληρώστε όλα τα πεδία !</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>barcode</tabstop>
  <tabstop>katigoria</tabstop>
  <tabstop>onoma</tabstop>
  <tabstop>mm</tabstop>
  <tabstop>expday</tabstop>
  <tabstop>posotita</tabstop>
  <tabstop>new_med</tabstop>
  <tabstop>katahorisi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
